### PR TITLE
Fix errcheck issues in server/channels/app/imaging/decode_test.go

### PR DESCRIPTION
**Pull request by samarth29jc**
#### Summary

Fixes errcheck issues in decode_test.go

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28946
JIRA: [MM-61255](https://mattermost.atlassian.net/browse/MM-61255)

#### Release Note

```release-note
NONE
```

[MM-61255]: https://mattermost.atlassian.net/browse/MM-61255?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/channels/app/imaging/decode_test.go [hunk 1]
```diff
@@ -53,7 +53,11 @@ func TestDecoderDecode(t *testing.T) {
 		imgFile, err := os.Open(imgDir + "/test.png")
 		require.NoError(t, err)
 		require.NotNil(t, imgFile)
-		defer imgFile.Close()
+
+		defer func() {
```
Comment (hanzei) [inline server/channels/app/imaging/decode_test.go:57]: Nit pick
```suggestion
		defer func() {
```

File: server/channels/app/imaging/decode_test.go [hunk 2]
```diff
@@ -222,4 +244,4 @@ func TestDecoderDecodeMemBounded(t *testing.T) {
 		release()
 		require.Empty(t, d.sem)
 	})
-}
+}
```
Comment (hanzei) [inline server/channels/app/imaging/decode_test.go:247]: Unrelated change

**Review by hanzei — CHANGES_REQUESTED** [review]
Nice work :+1:

File: server/channels/app/imaging/decode_test.go (hunk 1 above)
Comment (hanzei) [reply server/channels/app/imaging/decode_test.go]: Do you also want to fix the other occurrences?

**Review by calebroseland — APPROVED** [review]
LGTM, please fix lint

**Comment by samarth29jc** [comment]
> LGTM, please fix lint

DONE. 
Please Review and Merge

File: server/channels/app/imaging/decode_test.go [hunk 3]
```diff
@@ -207,7 +224,10 @@ func TestDecoderDecodeMemBounded(t *testing.T) {
 		imgFile, err := os.Open(imgDir + "/test.png")
 		require.NoError(t, err)
 		require.NotNil(t, imgFile)
-		defer imgFile.Close()
+
+		defer func() {
```
Comment (hanzei) [inline server/channels/app/imaging/decode_test.go:228]:
```suggestion
		defer func() {
```

**Pull request merged**